### PR TITLE
Use M70 to set label for the filament to load at the M600 Insert filament screen

### PR DESCRIPTION
**Pull request by Panayiotis-git**
A variation of the https://github.com/prusa3d/Prusa-Firmware/pull/4548. It repurposes the unused M70 command in the Prusa firmware to store a custom message, which is displayed by the M600 command on the load filament screen.

```
M70 Deep Blue
M600
```
The Insert filament screen will then show the custom text defined with the M70 command:
```
Insert filament
Deep Blue
and press the knob
```

### CODE REVIEW

File: Firmware/ultralcd.cpp [hunk 1]
```diff
@@ -5269,39 +5278,39 @@ static void lcd_main_menu()
...
-            } else if (IS_SD_PRINTING) {
-                MENU_ITEM_FUNCTION_P(_T(MSG_PAUSE_PRINT), lcd_pause_print);
-            }
+        if (usb_timer.running()) {
+            MENU_ITEM_FUNCTION_P(_T(MSG_PAUSE_PRINT), lcd_pause_usb_print);
+        } else if (IS_SD_PRINTING) {
+            MENU_ITEM_FUNCTION_P(_T(MSG_PAUSE_PRINT), lcd_pause_print);
         }
     }
+    }
```
Comment (gudnimg) [inline Firmware/ultralcd.cpp]: Something is off with indentation here.

File: Firmware/ultralcd.cpp [hunk 2]
```diff
@@ -5750,7 +5759,7 @@ void print_stop(bool interactive, bool unconditional_stop)
     // more move in this call! Any further move must happen inside lcd_print_stop_finish(), which is
     // called by the main loop one iteration later.
     if (unconditional_stop) {
-        UnconditionalStop();
+    UnconditionalStop();
```
Comment (gudnimg) [inline Firmware/ultralcd.cpp]: Indentation was correct before.

**Review by gudnimg — CHANGES_REQUESTED** [review]
Hi thanks for the PR. The only problem I see (at least on Github) is inconsistent indentation changes in the code. I commented on only two of them but there are more.

It could be a mix between spaces and tabs that makes the rendering different on Github.

**Comment by Panayiotis-git** [comment]
Something is really off...

I will resubmit the PR.

**Comment by gudnimg** [comment]
Much better, thanks!

**Review by gudnimg — APPROVED** [review]
Code looks good to me. I don’t see how it can go wrong. Should be tested on hardware though before merging, just in case.

**Comment by gudnimg** [comment]
I tested this on my MK3S+ and I didn't find any way to break things :)

* Sending only `M70` shows no message ✔️ 
* Sending `M70 "Hello World"` shows `"Hello World"` including the " characters ✔️ Exactly like M117
* Running `M600` the 2nd time (but not `M70`), no message is shown indicating it's reset properly. ✔️ 
* Sending `M70` with a very long string (more than 20 characters) is truncated correctly to the LCD line limit. ✔️ 

If no one objects to this PR, I would like to merge it.

**Comment by 3d-gussner** [comment]
Closing in favor of https://github.com/prusa3d/Prusa-Firmware/pull/4759

**Pull request closed without merging**